src/components/progress-bar/progress-bar.stories.ts

```diff
@@ -0,0 +1,39 @@
+export default {
+  title: 'Components/Progress Bar',
+  args: {
+    size: "m",
+    percentage: 50,
+    showLabel: false,
+
+  },
+  argTypes: {
+    percentage: {
+      control: { type: 'range', min: 0, max: 100 },
+    },
+    size: {
+      options: ['s', 'm'],
+      control: { type: 'radio' },
+    },
+  },
+};
+
+const DefaultTemplate = (args) =>
+  `<ifx-progress-bar percentage="${args.percentage}" size="${args.size}" show-label="${args.showLabel}" label="${args.percentage}%"></ifx-progress-bar>`;
```

Comment:
`label="${args.percentage}%"` why and where is this need needed? The prop `label` inside the component is not 
used anywhere.

Reply:
removed